games/s.yaml

```
@@ -1873,6 +1873,24 @@
   updated: 2022-08-20
   url: https://spelling-b.netlify.app
 
+- name: Spelunky Classic HD HTML5
+  originals:
+  - Spelunky Classic
+  type: clone
+  repo: 'https://github.com/yancharkin/SpelunkyClassicHDhtml5'
+  url: 'https://yancharkin.github.io/SpelunkyClassicHDhtml5/'
+  feed: 'https://github.com/yancharkin/SpelunkyClassicHDhtml5/releases/tag/1.1.7'
```

```suggestion
  feed: https://github.com/yancharkin/SpelunkyClassicHDhtml5/releases.atom
```
feed needs to be a valid RSS endpoint